src/App/analytics.js

```diff
@@ -1,7 +1,7 @@
 import GoogleAnalytics from 'electron-ga';
 import { reaction } from 'mobx';
 
-const CLIENT_ID = 'UA-22170471-17';
+const CLIENT_ID = 'UA-169320344-1';
```

Comment:
Sorry UA-169320344-1 can be used for testing. The current ID is the SIL instance, so should remain in the final code.

Reply:
Updated so it uses the dev one if in development mode; otherwise it uses the 'real' one.